cdap-ui/app/cdap/components/FormInputs/TextBox/index.tsx

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright © 2019 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+*/
+
+import React from 'react';
+import TextField from '@material-ui/core/TextField';
+import PropTypes from 'prop-types';
+import withStyles, { WithStyles } from '@material-ui/core/styles/withStyles';
+import ThemeWrapper from 'components/ThemeWrapper';
+
+const styles = (theme) => {
+  return {
+    ':hover': {
+      borderColor: '#cccccc',
```

Comment:
nit: The color should be used from theme. (`theme.palette.grey['50']`) + same for all color references in this style.

Reply:
Thanks! Grey['50'] is pretty dark - it looks more like grey['400']?

Reply:
SG!